prisma/schema.prisma

```diff
@@ -30,80 +18,67 @@ model ApprovedItemAuthor {
   @@index([approvedItemId])
 }
 
-// A prospect that is deemed good enough for saving in the corpus
-// and (optionally) making it to a Scheduled Surface.
 model ApprovedItem {
-  // fields
   id              Int                  @id @default(autoincrement())
-  externalId      String               @default(uuid()) @db.VarChar(255)
-  prospectId      String?              @db.VarChar(255)
-  url             String               @db.VarChar(500)
+  externalId      String               @unique @default(uuid()) @db.VarChar(255)
+  url             String               @unique @db.VarChar(500)
   title           String               @db.VarChar(255)
   excerpt         String               @db.Text
-  authors         ApprovedItemAuthor[]
   status          CuratedStatus        @default(CORPUS)
   language        String               @db.VarChar(2)
   publisher       String               @db.VarChar(255)
   imageUrl        String               @db.VarChar(500)
   topic           String?              @db.VarChar(255)
-  source          CuratedItemSource?
   isCollection    Boolean              @default(false)
-  isTimeSensitive Boolean              @default(false)
   isSyndicated    Boolean              @default(false)
   createdAt       DateTime             @default(now())
   createdBy       String               @db.VarChar(255)
   updatedAt       DateTime             @updatedAt
   updatedBy       String?              @db.VarChar(255)
-
-  // associated models
-  scheduledItems ScheduledItem[]
-
-  // indexes
-  @@unique([externalId])
-  @@unique([url])
+  prospectId      String?              @db.VarChar(255)
+  isTimeSensitive Boolean              @default(false)
+  source          CuratedItemSource?
+  authors         ApprovedItemAuthor[]
+  scheduledItems  ScheduledItem[]
 }
 
 model RejectedCuratedCorpusItem {
-  // fields
   id         Int      @id @default(autoincrement())
-  externalId String   @default(uuid()) @db.VarChar(255)
-  prospectId String?  @db.VarChar(255)
-  url        String   @db.VarChar(500)
+  externalId String   @unique @default(uuid()) @db.VarChar(255)
+  url        String   @unique @db.VarChar(500)
   title      String?  @db.VarChar(255)
-  topic      String?   @db.VarChar(255)
+  topic      String?  @db.VarChar(255)
   language   String?  @db.VarChar(2)
   publisher  String?  @db.VarChar(255)
-  // Can be multiple reasons. For the MVP, Snowplow and the frontend
-  // will be the only consumers of this data.
-  // Will be likely stored either as comma-separated values or JSON.
```

Comment:
The migration script stripped out all comments - perhaps for the better? As some of them appear to be outdated